xpack_security_get_user.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2012-2019 Oliver Eilhard. All rights reserved.
+// Use of this source code is governed by a MIT-license.
+// See http://olivere.mit-license.org/license.txt for details.
+
+package elastic
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+
+	"github.com/olivere/elastic/v7/uritemplates"
+)
+
+// XPackSecurityGetUserService retrieves a user by its name.
+// See https://www.elastic.co/guide/en/elasticsearch/reference/7.0/security-api-get-user.html.
+type XPackSecurityGetUserService struct {
+	client *Client
+	pretty bool
```

Comment:
The other standard fields are missing in this API as well, e.g. `filter_path`, `header` etc.

Reply:
Thank you for the review,
I have added the common options (based on https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html)